tasks/lint.js

```diff
@@ -7,6 +7,7 @@
 const { series, src } = require('gulp');
 
 // Load dependencies.
+/* eslint-disable-next-line import/no-unresolved */
```

Comment:
`eslint-plugin-import` [flags certain kinds of packages](https://github.com/import-js/eslint-plugin-import/issues/1810) as “unresolved” despite actually being resolvable. The maintainer does not seem motivated to address the issue due to this _technically_ being an issue with the `resolve` Node module. Our best option is to disable the rule for just this package.